classes/classes/BodyParts/BodyPart.as

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.CoC_Settings;
+import classes.Creature;
+import classes.internals.SimpleJsonable;
+
+public class BodyPart extends SimpleJsonable {
+	include "../../../includes/appearanceDefs.as";
```

Comment:
The move into a class was suggested as a first step.

From #507 :
Const groups that only use the class could then be moved out of appearanceDefs.as.

This probably was unclear, but essentially what you propose with 2.

1\. sounds like a cleaner way of doing checks, with the downside of having to create a lot of functions (another reason to move constants to the respective classes).

But it is probably for the better, as the code uses `int` instead of enums (which AS does not support), allowing people to do what ever they want. Also no type checking.